src/PAModel/Utility/DirectoryHelper.cs

```diff
@@ -29,17 +29,20 @@ public void DeleteAllSubdirs()
             {
                 Directory.CreateDirectory(_directory);
             }
-            foreach (var dir in Directory.EnumerateDirectories(_directory))
+            if (IsValidForDelete())
```

Comment:
Should we report an error and throw if the directory isn't valid for deletion?

Reply:
Yes, I think we should.  Because after unpack, they're going to repack, and we don't want to include a bunch of unrecognized files. 

The scenario here is that if they run it on the same git enabled folder each time, it should work ok. 

But if they run it on desktop, it should error.